cmd/kubeadm/app/cmd/certs.go

```diff
@@ -335,6 +335,14 @@ func getInternalCfg(cfgPath string, kubeconfigPath string, cfg kubeadmapiv1.Clus
 			internalcfg, err := configutil.FetchInitConfigurationFromCluster(client, nil, logPrefix, false, false)
 			if err == nil {
 				fmt.Println() // add empty line to separate the FetchInitConfigurationFromCluster output from the command output
+				// certificate renewal or expiration checking doesn't depend on a running cluster, which means the CertificatesDir
+				// could be set to a value other than the default value or the value fetched from the cluster.
+				// cfg.CertificatesDir could be empty if the default value is set to empty (not true today).
+				fmt.Printf("Overriding the default cert dir with the value from command line flag: %s", cfg.CertificatesDir)
```

Comment:
i think this should be a `klog.V(1).Infof()`
other than that LGTM.

/approve

Reply:
Done, thanks!